TombEngine/Specific/LevelCameraInfo.h

```diff
@@ -8,6 +8,7 @@ using std::string;
 
 struct LevelCameraInfo
 {
+	short Index = -1;
```

Comment:
What's the purpose of this?

Reply:
To locate a provided camera into the list of cameras in the level in a quicker way than go checking matching names.

By default is -1 what indicate the camera is not in the list (maybe a dynamic created camera?) but for the rest, they get their index during the level load, so when Lua sends a cameraObject, now it can get the Index value that identify that camera in the list, (and what later can be send to the Main camera during the PlayCamera function).

An alternative way is go setting the individual data into the Main Camera (the position, the roomnumber, flags and speed), but as Main camera code is already prepared to extract that data from a listed level camera, I opted to provide the lua camera like a similar input.

Reply:
Ok, I understand, nevertheless, I think that having index number which refers to a parent `std::vector` is a code smell. However, given that you will possibly refresh camera every frame, calling `std::find` every frame or using custom comparer is even worse, so let it be for now.